mentorship ios/Members/SendRequest.swift

```diff
@@ -36,27 +50,49 @@ struct SendRequest: View {
                         Text(LocalizableStringConstants.endDate)
                     }
 
-                    TextField(LocalizableStringConstants.notes, text: $notesText)
+                    TextField(LocalizableStringConstants.notes, text: $notes)
                 }
                 .padding(.vertical, DesignConstants.Padding.listCellFrameExpansion)
 
                 //send button
                 Section {
-                    Button(action: {}) {
+                    Button(action: sendRequest) {
                         Text(LocalizableStringConstants.send)
                     }
                 }
+                
+                //Activity indicator or error text
+                if membersModel.inActivity || !(membersModel.sendRequestResponseData.message ?? "").isEmpty {
```

Comment:
It's kind of unnecessary. Try optional (?) on sendRequestResponseData.message instead.As it is already initialised in MembersModel

Reply:
Or you can also use guard here

Reply:
I don't think guard can be used here. Because then we have to return a view in the else closure. Using if statements is recommended by Apple.

Reply:
I was just suggesting for this line: !(membersModel.sendRequestResponseData.message ?? ""). You are force unwrapping the value which always has a value i.e. ""
removing of ! would would also do the job.

Reply:
! is not to force unwrap, but for the bool. To make sure that the message string isn't empty.

Reply:
Oh! I think I was lost during the review. 😂